dtls.c

```diff
@@ -1157,10 +1185,18 @@ dtls_update_parameters(dtls_context_t *ctx,
   data += sizeof(uint16);
   data_length -= sizeof(uint16) + i;
 
+  if (ctx->h->get_cipher_suites != NULL) {
+    ctx->h->get_cipher_suites(ctx, &peer->session, &(config->cipher_suites));
```

Comment:
The parentheses around `config->cipher_suites` can be removed as the `->` operator takes precedence over `&`. (This could even lead to confusion as `&peer->session` does not use parentheses.)

Reply:
Done